src/main/java/net/spy/memcached/MemcachedConnection.java

```diff
@@ -745,8 +761,8 @@ private void updateNodeList(){
         lockForNodeUpdates.unlock();
       }
     }
-  
     conditionLock.lock();
+    isInitialClusterConfigApplied = true;
```

Comment:
Is it better to do `if (!isInitialClusterConfigApplied) isInitialClusterConfigApplied = true`?

Reply:
Updated